packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -21,11 +20,7 @@ const propTypes = {
   /**
    *  Value and optional Unit of Measure for the Observation Result
    */
-  result: PropTypes.oneOfType([
-    valueQuantityPropShape,
-    valueStringPropShape,
-    valueNullPropShape,
-  ]),
+  result: valueQuantityPropType,
```

Comment:
I know it's redundant but can we remove the re-usable prop types and add the prop types here directly? This would be easier from a documentation perspective so it's all face-up in the props table